components/encryption/src/main/java/org/cloudfoundry/credhub/services/EncryptionKeyRotator.java

```diff
@@ -38,9 +38,6 @@ public void rotate() {
     LOGGER.info("Starting encryption key rotation.");
     int rotatedRecordCount = 0;
 
-    final long startingNotRotatedRecordCount = encryptedValueDataService
-      .countAllByCanaryUuid(keySet.getActive().getUuid());
```

Comment:
Could this `countAllByCanaryUuid(` method be removed from the required interface since we don't use it anymore?

Reply:
I cancelled my accidental comments as reviewer, and now lost all my previous comments made in response to @bruce-ricard's comments. So reconstructing the lost comments from my memory here:

>Could this countAllByCanaryUuid( method be removed from the required interface since we don't use it anymore?

`countAllByCanaryUuid()` is indeed not invoked by anything. Even so, I'd rather keep the it. The method does a specific database search. I don't see great benefit in removing it.

> I feel like I would like, at the end of the rotation, a log of the type:

Are you suggesting to make another database query to get the kind of data that we are getting rid of in this PR? In any case, I do not think that having that additional log message would be very useful. The code is executed by scheduled task and will be executed again next time the scheduled task kicks in.